.babelrc

```diff
@@ -1,3 +1,8 @@
 {
-  "presets": ["@babel/preset-env"]
+  "presets": ["@babel/preset-env"],
+  "env": {
+    "test": {
+      "plugins": ["@babel/plugin-transform-runtime"]
```

Comment:
I'm not sure but jest uses babel as the default, so we need `regeneratorruntime` to use async/await. If we want to use the native async/await, we have to set `current` using `@babel/env`.

Reply:
We don't need this plugin for async/await:
1. It is inside `@babel/preset-env`
2. We use next branch, in node@8 it is built-in

Reply:
Jest don't use babel as the default, we need `babel-jest` package for this

Reply:
no, if we have `.babelrc`, jest uses `babel-jest`. see the documentation

Reply:
:+1: